clients/admin-ui/src/pages/_app.tsx

```diff
@@ -4,23 +4,31 @@ import '@fontsource/inter/700.css';
 
 import { ChakraProvider } from '@chakra-ui/react';
 import type { AppProps } from 'next/app';
-import { SessionProvider } from 'next-auth/react';
 import React from 'react';
+import { Provider } from 'react-redux';
 
-import { wrapper } from '../app/store';
+import store from '../app/store';
 import theme from '../theme';
 
 if (process.env.NEXT_PUBLIC_MOCK_API) {
   // eslint-disable-next-line global-require
   require('../mocks');
 }
 
+const SafeHydrate: React.FC = ({ children }) => (
+  <div suppressHydrationWarning>
+    {typeof window === 'undefined' ? null : children}
```

Comment:
Why is the hydration warning saying and doing? I'm guessing it's an annoying error message that we don't need while developing

Reply:
That's correct. Basically it's noting that what gets rendered on the server and what gets rendered on the client isn't the same, because we're no longer taking advantage of serverside rendering. This won't apply at all without a server, so the warning is meaningless.